benchmarks/buiter_et_al_2016_jsg/exp_2_low_resolution.prm

```diff
@@ -0,0 +1,210 @@
+#### This is the unstable wedge experiment 2
```

Comment:
If this parameter file is just a copy of the high-resolution version, you can derive it directly from this file and only include the changes (e.g., resolution). Let's talk later about how to do this.

Reply:
> If this parameter file is just a copy of the high-resolution version, you can derive it directly from this file and only include the changes (e.g., resolution). Let's talk later about how to do this.

Done.